### PR TITLE
Adding offline and configuration instructions

### PR DESCRIPTION
**Pull request by benjaminchodroff**
Apologies on the last PR mess, hope this one makes more sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
 # bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh)
 ```
 
+### V2Ray 組態
+1. 生成配置文件 [https://intmainreturn0.com/v2ray-config-gen](https://intmainreturn0.com/v2ray-config-gen/)
+2. 將配置文件 config.json放入 /usr/local/etc/v2ray/config.json
+
+### 離線安裝
+在具有網絡下載限制的環境中，我們建議：
```
Comment (IceCodeNew) [inline README.md:47]:
```suggestion
在具有網路下載限制的環境中，我們建議：
```

File: README.md [hunk 2]
```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
 # bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh)
 ```
 
+### V2Ray 組態
+1. 生成配置文件 [https://intmainreturn0.com/v2ray-config-gen](https://intmainreturn0.com/v2ray-config-gen/)
```
Comment (IceCodeNew) [inline README.md:43]:
```suggestion
1. 產生配置文檔 [https://intmainreturn0.com/v2ray-config-gen](https://intmainreturn0.com/v2ray-config-gen/)
```

File: README.md [hunk 3]
```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
...
+2. 將配置文件 config.json放入 /usr/local/etc/v2ray/config.json
+
+### 離線安裝
+在具有網絡下載限制的環境中，我們建議：
+1. 從GitHub.com下載fhs-install-v2ray存儲庫作為zip文件。
+2. 從以下位置下載v2ray-core發行zip文件  [https://github.com/v2fly/v2ray-core/releases](https://github.com/v2fly/v2ray-core/releases)
+3. 將兩個zip文件都上傳到您的服務器
```
Comment (IceCodeNew) [inline README.md:50]:
```suggestion
3. 將兩個 zip 文檔都上傳到您的伺服器
```

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
1. 建议使用您熟悉的中文变体撰写内容，并通过 OpenCC 自动转换为另一变体的内容。目前添加的内容用繁体字讲简体中文，任谁看了都难受。
2. 为了避免 Readme 文档过长让人没有耐心读完，建议将这部分内容移到 Wiki 中。
3. 为了避免增加您的工作，同时也是因为修改 Wiki 不需要审核，难以把控质量。我建议您继续在这一 PR 上完善内容，最后由我来将指南移到 Wiki 上。

1. It is recommended that you write the instructions in the Chinese variant you are familiar with and convert it to another variant by using the OpenCC service. The current change is difficult for any Chinese users to understand.
2. To avoid the Readme being too long for people to read through, I recommend moving this section to the Wiki.
3. To avoid putting an extra workload on you and better control this instruction's quality (as changing the Wiki does not require review), I suggest that you continue to work on this PR, and I will eventually move the guide to the Wiki.

**Comment by IceCodeNew** [comment]
image
image

https://opencc.byvoid.com/

File: README.md (hunk 2 above)
Comment (kslr) [reply README.md:43]: 这个生成器很多年没有更新了

**Comment by kslr** [comment]
我认为只需要在 readme 列出命令列表就够了。
如果要写一份 指南 不妨用自己熟悉的语言，而不是通过翻译形式，因为这需要长期的维护。

**Comment by IceCodeNew** [comment]
> 我认为只需要在 readme 列出命令列表就够了。
> 如果要写一份 指南 不妨用自己熟悉的语言，而不是通过翻译形式，因为这需要长期的维护。

附议。不过我个人把中文不同变体之间互相转换的问题看作翻译以外的形式，OpenCC 也是相当成熟的解决方案了，比起两边更新不同步来说，一份大致上阅读不造成障碍的自动转换成的文章我觉得是各方面都更优的选择。

**Comment by benjaminchodroff** [comment]
Thank you for the review. I have incorporated the suggested changes. I hope these suggestions help others quickly adopt v2ray. My biggest challenges in implementing v2ray were finding a reliable installation and configuration. I hope this project can continue to point people in the right direction.

File: README.md [hunk 4]
```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
 # bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh)
 ```
 
+### V2Ray 組態
+1. 產生配置文檔 [https://intmainreturn0.com/v2ray-config-gen](https://intmainreturn0.com/v2ray-config-gen/)
```
Comment (kslr) [inline README.md:43]: This config generation is wrong, you should not add it

File: README.md [hunk 5]
```diff
@@ -39,6 +39,18 @@ installed: /etc/systemd/system/v2ray@.service
...
+2. 將配置文件 config.json放入 /usr/local/etc/v2ray/config.json
+
+### 離線安裝
+在具有網路下載限制的環境中，我們建議：
+1. 從GitHub.com下載fhs-install-v2ray存儲庫作為zip文件。
+2. 從以下位置下載v2ray-core發行zip文件  [https://github.com/v2fly/v2ray-core/releases](https://github.com/v2fly/v2ray-core/releases)
+3. 將兩個 zip 文檔都上傳到您的伺服器
+4. 解壓縮fhs-install-v2ray存儲庫zip文件
+5. 運行安裝，並將其指向本地v2ray-core zip文件：```sudo bash install-release.sh --local /path/to/v2ray-linux-64.zip```
```
Comment (kslr) [inline README.md:52]: 1. Chinese and English need spaces 
2. Which files to upload? 
3. ...

I think this is more suitable for a personal blog

**Review by kslr — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (benjaminchodroff) [reply README.md:43]: As an alternative, I could include a template config.json.template file with instructions/script on how to generate a secure UUID. Would this be better and acceptable?

File: README.md (hunk 5 above)
Comment (benjaminchodroff) [reply README.md:52]: 1. Can you please suggest the spaces? Chinese is not my native language, my apologies
2. Many users are struggling to find documentation on how to set up v2ray with a simple but secure configuration. I would suggest this is something that should be included with the installation.

**Review by lixianglan1 — APPROVED** [review]

**Pull request closed without merging**